app/assets/javascripts/components/search/SearchResults.js

```diff
@@ -13,9 +13,10 @@ export const SearchResults = ({ isEnabled, searchResults, closeSearch, projectBo
 
   return (
     <Column
+      title={`"${projectBoard.search.keyWord}"`}
```

Comment:
Are these double-quotes correct?

Reply:
Yes, the title of this column needs of double-quotes for show the keyword search.